core/modules/simpletest/tests/form.test

```diff
@@ -66,7 +66,7 @@ class FormsTestCase extends BackdropWebTestCase {
     $elements['file']['empty_values'] = $empty_strings;
 
     // Regular expression to find the expected marker on required elements.
-    $required_marker_preg = '@<label.*<abbr class="form-required" title="This field is required\.">\*</abbr></label>@';
+    $required_marker_preg = '@<label.*<abbr class="form-required" title="This field is required\." aria-hidden="true">\*</abbr>.*</label>@';
```

Comment:
Should we make this more explicit, by also adding the newly-introduced `<span>` to the regex?:

```suggestion
    $required_marker_preg = '@<label.*<abbr class="form-required" title="This field is required\." aria-hidden="true">\*</abbr><span class="element-invisible">.*</span></label>@';
```

Reply:
Sure, let's try that.